docs/site/Model.md

```diff
@@ -190,9 +188,198 @@ class Product extends Entity {
 }
 ```
 
-Additionally, the model decorator is able to build the properties object through
-the information passed in or inferred by the property decorators, so the
-properties key value pair can also be omitted.
+However, the model decorator in lb4 is not exactly the same as what it is in
+lb3. For example, in
+[lb3](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html) we can
+pass in a model definition syntax in model decorator, such as properties,
+options, relation etc. But not all these entries are available in lb4 model
+decorator:
+
+In lb4, we only support `properties` and `setting` in the ModelDefinitionSyntax
+for now. Those `top-level properties` in lb3 now are passed in `settings`.
+[Options](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#options)
+in lb3 doesn't have the mapping feature in lb4 yet.
+
+As for entries in `settings`, lb4 supports these built-in entries for now:
+
+#### Available Entries of Settings
+
+  <table>
+  <thead>
+  <tr>
+    <th width="160">Property</th>
+    <th width="100">Type</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+
+  <tbody>
+  <tr>
+  <td><code>base</code></td>
```

Comment:
AFAIK, `base` is not supported in LB4. We use JavaScript class inheritance instead.

Reply:
In [model-builder file](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/model-builder.js#L157), it sets up the base here when I run the debug tool. That's why I think it's available. Could you explain more about this part?

Reply:
Sure. You are right that juggler's model-builder does support `base` setting.

Here is how model inheritance should be used in LB4:

```ts
@model()
class BaseUser extends Entity {
  // properties and methods used by all models
}

@model()
class Customer extends BaseUser {
  // properties specific to customers
}

@model()
class Administrator extends BaseUser {
  // properties specific to site admins
}
```

I would expect that under the hood, the code converting LB4 model to juggler model definition should take care of model inheritance and configure `base` setting as needed. See [`definePersistedModel`](https://github.com/strongloop/loopback-next/blob/2fa5df67181cdcd23a5dce90c9c640fe75943cb8/packages/repository/src/repositories/legacy-juggler-bridge.ts#L135-L187). (I don't see any such code in that method though.)

To be honest, I am not sure if we ever tested model inheritance in LB4, it may not even work properly.

I also don't know what happens when a LB4 model explicitly configures the `base` setting, this is where a unit/integration test would be very useful.